libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightConfigurationHeader.tsx

```diff
@@ -0,0 +1,40 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { Button, Typography } from "@gooddata/sdk-ui-kit";
+import { FormattedMessage } from "react-intl";
+import { SelectedItemType } from "../../common/configuration/types";
+
+interface IInsightConfigurationProps {
+    selectedItemType: SelectedItemType;
+    handleConfigurationHeaderClick: () => void;
+    onCloseButtonClick: () => void;
+}
+
+export const InsightConfigurationHeader: React.FC<IInsightConfigurationProps> = ({
+    selectedItemType,
+    handleConfigurationHeaderClick,
+    onCloseButtonClick,
+}) => {
+    return (
+        <div className="configuration-panel-header">
+            <Typography
+                tagName="h3"
+                className="configuration-panel-header-title clickable"
+                onClick={handleConfigurationHeaderClick}
+            >
+                <i className="gd-icon-navigateleft" />
+                <FormattedMessage
+                    id={
+                        selectedItemType === "configuration"
```

Comment:
Will this be picked up by the i18n validator? Please try typing some nonsense string for the id and run `yarn validate!. If it passes, this does not work and you will have to rewrite this to st. like

```jsx
selectedItemType === "configuration" 
    ? <FormattedMessage id="configurationPanel.title" /> 
    : <FormattedMessage id="configurationPanel.interactions" />

Reply:
It was not validating correctly, so I changed that.